examples/log_config.yaml

```diff
@@ -0,0 +1,25 @@
+version: 1
+disable_existing_loggers: false
+formatters:
+  pytroll:
+    format: '[%(asctime)s %(levelname)-8s %(name)s] %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: pytroll
+    stream: ext://sys.stdout
+  monitor:
+    (): pytroll_monitor.op5_logger.AsyncOP5Handler
```

Comment:
This could use some explanation/documentation.  This is the first time I have heard of `pytroll_monitor`.  Is this a new dependency?

Reply:
Indeed, this should be here, just regular config items should be used

Reply:
The above yaml file is an example usage of this optional plugin. What and where do you want explanations?
With pytroll_monitor is is possible to use the log-messaging to communicate with a NAGIOS type central monitoring - "passive monitoring"

Reply:
To me the `monitor:` block is just confusing.  I don't know what `():` means and I didn't know what `pytroll_monitor` is.  Apparently it's a logging handler?  But there is no formatter defined for this handler.  I might add that I do usually get confused with the Python logging system so it may just be me.  At least for the console handler I recognise most keys as keyword arguments so I can guess what's going on.  

@mraspaud Did you mean should or shouldn't?

Reply:
I meant "shouldn't" at the time, but I'm realizing now that this is nice way to introduce pytroll_monitor. The syntax of this log config is straight out from the python logging documentation, so I'm not sure what more we could do here. Maybe link to https://docs.python.org/3/library/logging.config.html ?
Regarding the formatter, not providing one equates to using the default one iirc.

Reply:
I leave as is!
I don't see an obvious place to put the doc link, and people could easily find that themselves.